src/OpenApi.hs

```diff
@@ -0,0 +1,19 @@
+module OpenApi
+  ( module OpenApi.Types
+  , loadSpecFile
+  )
+where
+
+import Data.Aeson
+import Data.ByteString.Lazy (ByteString, readFile)
+import OpenApi.Prelude
+import OpenApi.Types
+import System.FilePath (FilePath)
+
+loadSpecFile :: (MonadFail m, MonadIO m) => FilePath -> m Specification
+loadSpecFile = loadSpec <=< (liftIO . readFile)
+
+loadSpec :: (MonadFail m, MonadIO m) => ByteString -> m Specification
```

Comment:
@mbj what is your convention for when to break functions out into top-level functions or have them as private functions? In this specific case, if `loadSpec` was a private function you could remove the `(MonadFail m, MonadIO m) =>` declaration.

Reply:
I think I originally had an export for `loadSpec` which I later dropped. And the toplevel is an artifact.